HelpSource/Classes/ServerTreeView.schelp

```diff
@@ -57,7 +57,7 @@ a.stop
 [a, c].do { |v| v.close }
 
 // Open a plotTree window:
-s.plotTree
+s.plotTree // This changes also the interval of the ServerTreeView instances assinge to variabled since there is currently only one updater for this class.
```

Comment:
minor change suggested:
`This also changes the interval of the ServerTreeView instance assigned to a variable, since`...

Reply:
Thanks, done in https://github.com/supercollider/supercollider/pull/6834/commits/bf4cc47da33fa088117afc433197068e265af4db